README.md

```diff
@@ -140,6 +141,7 @@ controller_server:
       vy_std: 0.2
       wz_std: 1.0
       vx_max: 0.5
+      vx_min: -0.2
```

Comment:
Make this -0.35 here + in defaults.

It might seem arbitrary, but it is not. For some reason the difference between 0.3 and 0.2 is the difference between if the robot will try to do backing up in confined maneuvers, which feels much more natural.

The +0.05 is just to add a little more leeway there.

Reply:
done